src/components/reusable/CardGallery.js

```diff
@@ -0,0 +1,90 @@
+import { React, useState } from "react";
+import Card from "./Card";
+
+const CardGallery = () => {
+  const [selectedIndicator, setSelectedIndicator] = useState("MACD");
+
+  const handleChange = (event) => {
+    setSelectedIndicator(event.target.value);
+  };
+
+  const indicators = [
+    {
+      name: "MACD",
+      description:
+        "Moving average convergence divergence (MACD, or MAC-D) is a trend-following momentum indicator that shows the relationship between two exponential moving averages (EMA's) of a security’s price. The MACD is calculated by subtracting the 26-period exponential moving average (EMA) from the 12-period EMA.",
+      img: null,
```

Comment:
Do you think you'll ever want to add images here? I f not the img field could just be removed.